stdlib/public/core/DictionaryStorage.swift

```diff
@@ -180,21 +186,21 @@ extension _EmptyDictionarySingleton: _NSDictionaryCore {
 }
 #endif
 
-extension _RawDictionaryStorage {
+extension __RawDictionaryStorage {
   /// The empty singleton that is used for every single Dictionary that is
   /// created without any elements. The contents of the storage should never
   /// be mutated.
   @inlinable
   @nonobjc
-  internal static var empty: _EmptyDictionarySingleton {
+  internal static var empty: __EmptyDictionarySingleton {
     return Builtin.bridgeFromRawPointer(
       Builtin.addressof(&_swiftEmptyDictionarySingleton))
   }
 }
 
 @usableFromInline
 final internal class _DictionaryStorage<Key: Hashable, Value>
```

Comment:
This one didn't need to be renamed because it's new? (And the same for the Set one.)

Reply:
Turns out yes, but I (correctly) missed it by accident, so that's bad.